run-tests.js

```diff
@@ -322,6 +322,23 @@ async function main() {
 
       const shouldRecordTestWithReplay = process.env.RECORD_REPLAY && isRetry
 
+      const traceEnv =
+        process.env.DATADOG_API_KEY && process.env.DATADOG_TRACE_NEXTJS_TEST
+          ? {
+              DD_API_KEY: process.env.DATADOG_API_KEY,
+              DD_CIVISIBILITY_AGENTLESS_ENABLED: 'true',
+              DD_ENV: 'ci',
+              DD_SERVICE: 'nextjs',
+              NODE_OPTIONS: !!process.env.DATADOG_API_KEY
+                ? '-r dd-trace/ci/init'
+                : undefined,
+            }
+          : {}
+
+      if (!!traceEnv.DD_API_KEY) {
```

Comment:
```suggestion
      if (traceEnv.DD_API_KEY) { 
```

The `!!` is redundant

Reply:
Thanks, updated.